examples/videocomposer/vc/annotator/depth/midas.py

```diff
@@ -200,7 +202,9 @@ def construct(self, *xs):
             x = self.layer2(xs[0] + self.layer1(xs[1]))
         size = x.shape[2:]
         size = [s * 2 for s in size]
-        x = ops.interpolate(x, size=size, mode="bilinear", align_corners=True)
+        ori_dtype = x.dtype
+        x = ops.interpolate(x.to(ms.float32), size=size, mode="bilinear", align_corners=True)
```

Comment:
See examples/stable_diffusion_xl/gm/modules/diffusionmodules/model.py, L36.
According to huiyao, ops.interpolate runs on aicpu, which is too slow.

Reply:
Huiyao uses `ops.ResizeNearestNeighbor`. I heard `ops.ResizeNearestNeighbor` is not supported by 910B ai core. How about `ops.ResizeBilinearV2`?

Reply:
it depends on MS version, for MS2.2 0907, it supports ops.ResizeNearestNeighbor but not for MS 2.1